functions/util.js

```diff
@@ -8,3 +8,42 @@ export function shuffleArray(array) {
   }
   return array;
 }
+
+export function getTimeAgo(timestring) {
```

Comment:
Instead of printing the date when a quiz was taken this gets how long ago it was